widgets/common/IncompatiblePointDistribution.tsx

```diff
@@ -179,4 +182,349 @@ class IncompatiblePointDistribution extends Component<IncompatiblePointDistribut
     );
   }
 }
-export default IncompatiblePointDistribution;
+
+
+type BarChartSelectorProps = {
+  selectedChart: string
+  setSelectedChart: Function
+}
+
+class BarChartSelector extends Component<BarChartSelectorProps, null> {
+
+  render () {
+    let getBlockClass  = (regionName: string) => {
+      if (this.props.selectedChart == regionName) {
+        return "chart-selector-row-block-selected";
+      } else {
+        return "chart-selector-row-block";
+      }
+    }
+
+    let getBgColor = (chartName: string) => {
+      if (this.props.selectedChart == chartName) {
+        return "rgba(0, 120, 212, 1)";
+      }
+
+      return "rgba(255, 255, 255, 1)";
+    }
+
+    let getTextColor = (chartName: string) => {
+      if (this.props.selectedChart == chartName) {
+        return "rgba(255, 255, 255, 1)";
+      }
+
+      return "rgba(0, 0, 0, 1)";
+    }
+
+    return (
+      <div className="chart-selector-row">
+        <div className={getBlockClass("model-accuracy")} onClick={() => this.props.setSelectedChart("model-accuracy")}>
```

Comment:
Use PrimaryButton and DefaultButton from FluentUI here. Those will automatically match the styles in the mockups and also have full accessibility features.

Reply:
I tried to use them. But the issue I had was that they were too big. And then there was the issue of aligning everything so that it fit, i.e. with the legend and all that. So I opted to style these.

Is there a way to do all that with the PrimaryButton and DefaultButton?

Reply:
Fluent UI has a way of applying CSS it with a "styles" prop. You might also be able to do it in CSS with a specific CSS selector like `.chart-selector-row > .ms-Button`.

Here's some code from my branch that shows how you could create a highly styled button in Typescript:
`<DefaultButton iconProps={{iconName: "ZoomIn"}} text="zoom in" styles={{root: {paddingLeft: "9px", paddingRight: "9px", marginRight: "12px"}}}/>`

The alignment and size issues will probably be a problem until the widget is widened to 1000px width.

Reply:
Great. Thanks. Will try this out.